app/tag-management/delete-tag/delete-tag.component.html

```diff
@@ -0,0 +1,10 @@
+<md-card class="default-card">
+	<md-card-title>{{ 'delete tag' | translate }}</md-card-title>
```

Comment:
In all other components, we have an extra md-card for the title and using `h1` as headline. Please do it like that for your components as well to be consistent:
```
<md-card class="default-card">
   <h1>Title</h1>
</md-card>
```